examples/pipelines/simple_pipeline/components/image_filtering/requirements.txt

```diff
@@ -1,3 +1,3 @@
-git+https://github.com/ml6team/fondant.git@d697be951fa33faa0baa3625cf3cb04d6800834a
+git+https://github.com/ml6team/fondant.git
```

Comment:
Since we squash merge, any commit hash part of a PR will actually not become part of `main`, so it's not ideal to use them as a reference.

I think we should always keep these examples in line with main, so I would propose to unfreeze the fondant version in their requirements.

Reply:
I think those are mainly used for testing the functionality of your branch code. Right now the workflow is a bit annoying to do so: you have to push your code and then reference the hash so that the component can rebuild with the latest version. Wondering if there isn't a better way to do. @NielsRogge any thoughts?

Reply:
Once we 'actually open source', the examples on main should always work though. For components to share, the version can still be fixed.

Reply:
Yes for now we probably need to include the commit hash when trying out new functionality, but when merging a PR it should always be updated to main.